packages/headless/package.json

```diff
@@ -47,7 +47,7 @@
     "@reduxjs/toolkit": "^1.5.0",
     "@types/pino": "^6.3.4",
     "@types/redux-mock-store": "^1.0.2",
-    "coveo.analytics": "^2.18.45",
+    "coveo.analytics": "^2.18.50",
```

Comment:
I opened a PR too[ bumping the version](https://github.com/coveo/ui-kit/pull/1374/files#diff-b49fb4b1d19b4b6568e5ff36ec318c7410ab21ce4405d21930510332d64c3905R50) to get the static filter analytics actions 🤷

Reply:
Okay, closing this